server/processor.go

```diff
@@ -11,15 +12,22 @@ import (
 	"github.com/pkg/errors"
 )
 
-const postsPerMinuteLimit = 500
-const processingInterval = 1 / postsPerMinuteLimit * time.Minute
+const (
+	postsPerMinuteLimit = 500
+	processingInterval  = 1 / postsPerMinuteLimit * time.Minute
+
+	// Message templates for moderation notifications
+	channelNotificationTemplate = "_A post with potentially offensive content was flagged and removed._"
```

Comment:
Do we plan to add internationalization in future?

Reply:
Yes, we probably will :slightly_smiling_face: